doc/source/whatsnew/v1.5.1.rst

```diff
@@ -92,6 +92,7 @@ Fixed regressions
 
 Bug fixes
 ~~~~~~~~~
+- Bug in :func:`get_dummies` with default ``dtype`` being ``uint8`` - the default ``dtype`` is now changed to ``bool`` (:issue:`45848`)
```

Comment:
@MarcoGorelli could you remind me what was the reasoning this API change was okay for 1.5.1 instead of 1.6.0/2.0?

Reply:
just that it might have been small enough to fit in to the 1.5.1 fixes which hadn't made it in time for 1.5.0 - no objections to waiting though

Reply:
I'd be slightly more comfortable for this to go into 1.6/2.0 since on the surface it appears more like an API change to me. Though I understand it can fix subsequent overflow bugs.

Otherwise if other are set for this in 1.5.1, I would suggest calling this out more in an `API Change` section.

Reply:
no strong feelings, could always take it to Wednesday's call to double-check, I doubt 1.5.1 would happen before then anyway?

Reply:
Agreed with @mroeschke 1.6 feels more "right"

Reply:
So move it to v1.6 whatsnew?

Reply:
yes please